interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -258,7 +258,7 @@ function display_PRIOR_section($zone, $orig_id, $id_to_show, $pid, $report = '0'
         }
         ?>
```

Comment:
There is the outdated `left` and `right` classes in this file that need to be replaced with `float-left` and `float-right`

Reply:
Also has a lot of outdated code as well that I recommend re-going over